src/firefox/index.js

```diff
@@ -99,6 +99,11 @@ export interface FirefoxProcess extends events$EventEmitter {
   kill: Function;
 }
 
+export interface IProfileFinder {
```

Comment:
This flow type is currently unused and it should be used for the `defaultCreateProfileFinder` return value type annotation (we should also move it near that function, that is where it is going to be used).

Also, the type signatures are wrong: both `hasProfileName` and `getPath` take a string as parameter
(and so they should be `hasProfileName(string): Promise<boolean>`, and `getPath` needs the same kind of fix).